docs_nnx/guides/checkpointing.md

```diff
@@ -1,17 +1,35 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+---
+
 # Save and load checkpoints
 
-Flax does not actively maintain a library for saving and loading model checkpoints to disk. We recommend you to use external libraries like [Orbax](https://orbax.readthedocs.io/en/latest/index.html) to do it.
+This guide demonstrates how to save and load Flax NNX model checkpoints with [Orbax](https://orbax.readthedocs.io/). You will learn how to:
```

Comment:
In the future we *might* change this to use a built-in checkpointing library of JAX, or maybe not. 
Regardless, I hoped this document could focus less on the specific checkpointing library and more on the Flax NNX side of API. Maybe we can keep the original phrasing order?

Reply:
Thanks @IvyZX As discussed with you:

```
# Save and load checkpoints

This guide demonstrates how to save and load Flax NNX model checkpoints with [Orbax](https://orbax.readthedocs.io/).

> **Note:** The Flax team does not actively maintain a library for saving and loading model checkpoints to disk. Therefore, it is recommended you use external libraries like [Orbax](https://orbax.readthedocs.io/en/latest/index.html) to do it.

In this guide you will learn how to:
```